test/functional/feature_block.py

```diff
@@ -7,7 +7,9 @@
 import struct
 import time
 
-from test_framework.blocktools import create_block, create_coinbase, create_tx_with_script, get_legacy_sigopcount_block
+from test_framework.blocktools import (
+    create_block, create_coinbase, create_tx_with_script, get_legacy_sigopcount_block,
+    MAX_BLOCK_SIGOPS)
```

Comment:
Not used in this file?

Reply:
https://github.com/bitcoin/bitcoin/pull/14457/files/1483640cf1f106c62c0fbe026da8d60e1992c91f#diff-61418630f672af16f132a5c0b56fb6fbR174

Reply:
https://github.com/bitcoin/bitcoin/blob/1483640cf1f106c62c0fbe026da8d60e1992c91f/test/functional/feature_block.py#L244